react/react-training/src/app/screens/Dashboard/screens/Profile/styles.scss

```diff
@@ -6,3 +6,151 @@
   font-size: $title-size;
   text-align: center;
 }
+
+.card {
+  align-content: space-around;
+  background-color: #757575;
+  border-radius: 10px;
+  box-shadow: 3px 3px 20px rgba(0, 0, 0, .5);
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: auto;
+  padding: 10px;
+  width: 500px;
+}
+
+.framework, .info {
+  flex: 1;
+}
+
+.framework {
+  text-align: center;
+}
+
+.photo {
+  border-radius: 50%;
+  height: 150px;
+  width: 150px;
+
+  -webkit-animation: fadein 2s; /* Safari, Chrome and Opera > 12.1 */
```

Comment:
This prefix isn't necessary because the sass compiler assigns it, so delete it all